README.md

```diff
@@ -137,6 +137,15 @@ options:
   --port PORT                   service port
 ```
 
+## Efficient retrieval with ANN search
+
+One important aspect of deploying recommender model is efficient retrieval via Approximate Nearest Neighor (ANN) search in vector space. Although, this is not the main focus of Cornac, we try our best to integrate several open-source frameworks out-of-the-box. [This example](tutorials/ann_hnswlib.ipynb) demonstrates how ANN search will work seamlessly as other recommender models. 
```

Comment:
"demonstrates how ANN search will work seamlessly as other recommender models" it sounds like ANN-search is another recommender model. How about the following rephrasing: "demonstrates how we can leverage ANN search with a recommender model (e.g., MF)"

Reply:
Well, that's the whole point here. We want to make the wrapper works as a recommender so all the APIs including save/load/recommend are consistent. This will help integrating any utils we're developing to support model deployment easier. We don't want to maintain one for a model and another one for ANN search.

Reply:
The warper aspect is great, we need it for standardization. It just sounds confusing to call the ANN a recommender given that it's not a standalone model. Maybe we just need some rephrasing to make it clear, and it's fine for the wrapper to seat under cornac/models. Maybe I am missing something though haha

Reply:
I don't want to be so philosophical here lol. To me, a recommender doesn't have to be a model, it could be any heuristics that's being able to recommend items. Just that we named the folder `models` so it might contradicts a bit. My intent is to emphasize that ANN will work similarly to other model-based recommender in Cornac, in terms of API usage to do recommendation. Anw, we can rephrase this part to make it clearer.

Reply:
updated